dimension-reduction/pca_example.Rmd

```diff
@@ -80,16 +91,26 @@ Label the data points based on their genotype group.
 
 ```{r}
 # Make a scatterplot with ggplot2
-ggplot(pca.df, aes(x = PC1, y = PC2, color = `genotype/variation`)) +
+pca.plot <- ggplot(pca.df, aes(x = PC1, y = PC2, color = subgroup)) +
   geom_point() + 
   theme_classic()
+
+# Print out plot here
+pca.plot
+```
+
+## 4) Save plot to png.
+
+```{r}
+# Save as a png
+ggsave(file.path("plots", "pca_scatterplot.png"))
 ```
 
 ## 5) Save PC scores to tsv file.
 
 ```{r}
 pca.df %>% as.data.frame() %>% 
-  readr::write_tsv(file.path("results", "PCA_scores.tsv"))
+  readr::write_tsv(file.path("results", "pca_scores.tsv"))
```

Comment:
I suspect this file name change is unintentional. There is also no `results/pca_scores.tsv` file being added here.